dbc2val/test/test_dbc/test1_2.kcd

```diff
@@ -1,5 +1,24 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<NetworkDefinition xmlns="http://kayak.2codeornot2code.org/1.0">
+<!--
+Copyright (c) 2023 Robert Bosch GmbH
```

Comment:
@SebastianSchildt  - for new files, do we want this form or the more generic "Copyright (c) 2023 Contributors to the Eclipse Foundation"

Reply:
good point :+1: 

the other Eclipse projects I am involved in usually take the more generic approach and let contributors add their affiliation to the NOTES.md file ...

Reply:
We have for some time now been using the generic "Copyright (c) 2023 Contributors to the Eclipse Foundation" for all new files, as that seems to be the current best practice.
It is however currently neither enforced nor written down as a decision somewhere, we just went where the river took us.

Reply:
Ok, I will adapt the PR accordingly.